ts/indicators/dpo.src.ts

```diff
@@ -0,0 +1,197 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        class DPOIndicator extends SMAIndicator {
+            public data: Array<DPOIndicatorPoint>;
+            public options: DPOIndicatorOptions;
+            public pointClass: typeof DPOIndicatorPoint;
+            public points: Array<DPOIndicatorPoint>;
+            public yData: Array<Array<number>>;
+            public getValues(
+                series: Series,
+                params: DPOIndicatorParamsOptions
+            ): (boolean|IndicatorValuesObject);
+        }
+
+        interface DPOIndicatorOptions extends SMAIndicatorOptions {
+            params?: DPOIndicatorParamsOptions;
+        }
+
+        interface DPOIndicatorParamsOptions extends SMAIndicatorParamsOptions {
+            period: number;
```

Comment:
This should be optional.

Reply:
Ok, so then it should be cast as any in the top of the implementation? (period is optional in SMAIndicator)

Reply:
Fixed